powerstore/resource_snapshot.go

```diff
@@ -88,7 +90,8 @@ func (r *resourceVolumeSnapshot) Schema(ctx context.Context, req resource.Schema
 				},
 			},
 			"expiration_timestamp": schema.StringAttribute{
-				Required:            true,
+				Optional:            true,
```

Comment:
Can we remove the timestamp later once assigned?

Reply:
Ideally it should be the case but for now we have not implemented update yet, so it will be taken care as a part of it.